src/app/public/modules/vertical-tabset/vertical-tabset-group.component.html

```diff
@@ -11,34 +11,14 @@
     [tabIndex]="-1"
   >
     {{ groupHeading }}
-    <sky-chevron *ngIf="themeName === 'default'"
+    <sky-chevron
```

Comment:
Have we double checked this change with design? This isn't a 1:1 change. In modern - the chevron points to the right and then changes to a chevron pointing down when the vertical tab is selected. This change would change it to be an up and down chevron. 

We use the up and down chevron in default theme so my assumption is that this was a deliberate modern theme change and so I wouldn't want to change that without design approval.

Reply:
That being said - now that chevron supports modern theme it could lead to the down chevron looking different than the one pointing right. @Blackbaud-ToddRoberts or @Blackbaud-AdamFunderburk any thoughts on this?

Reply:
I reverted back to the chevron right icon.